distributed/distributed.yaml

```diff
@@ -195,6 +195,8 @@ distributed:
         key: null
         cert: null
 
+  diagnostics:
+    nvml: True
```

Comment:
Since this is GPU specific, we might want to put this option in it's own section like `rmm` and `ucx`

https://github.com/dask/distributed/blob/0c154324bdfabfc28b1a773d878c759ad35252cc/distributed/distributed.yaml#L228-L237

Reply:
I don't mind moving it there if you prefer, but it seems like the general feel about those is that they have been a mistake to be outside of the `distributed` schema. For example https://github.com/dask/distributed/issues/4904 is attempting to move `ucx` into `distributed` to comply with the general config schema.